Corona-Warn-App/src/main/res/values-bg/contact_diary_strings.xml

```diff
@@ -154,10 +154,10 @@
     <!-- XACT: Add Location Button (description for screen readers) -->
     <string name="accessibility_day_add_location">"Добавяне на място"</string>
 
-    <!-- XBUT: Option - person encounter - below 15 Min -->
-    <string name="contact_diary_person_encounter_duration_below_15_min">"По-малко от 15 мин"</string>
-    <!-- XBUT: Option - person encounter - above 15 Min -->
-    <string name="contact_diary_person_encounter_duration_above_15_min">"Повече от 15 мин"</string>
+    <!-- XBUT: Option - person encounter - below 10 Min -->
+    <string name="contact_diary_person_encounter_duration_below_10_min">"По-малко от 10 мин"</string>
```

Comment:
hm.. I'm not sure if this won't create merge conflicts. AFIK we usually update only DE strings (+ EN if it's necessary)

Reply:
how should a mergeconflict occur?

Reply:
XML files for different languages are generated automatically by a translation tool. If you update these files also manually in the meantime, translation delivery PR would get a conflict and somebody has to check changes in these files. At least according to what I heard.

Reply:
@BMItr  afaik @jurajkusnier  is right renaming resource Ids could lead to some problems in the translation tool, 
I would suggest that you add new key for the required translation in this case 
`contact_diary_person_encounter_duration_below_10_min` in `DE`, `EN` since it is new 
and delete the old key in this case 
`contact_diary_person_encounter_duration_above_15_min` 
ONLY  in `DE` translation tool will take care of the rest locales